Bloxstrap/Integrations/ActivityWatcher.cs

```diff
@@ -212,6 +214,21 @@ private void ReadLogEntry(string entry)
             {
                 // We are not confirmed to be in a game, but we are in the process of joining one
 
+                if (entry.Contains(GameJoinLoadTimeEntry))
+                {
+                    Match match = Regex.Match(entry, GameJoinLoadTimeEntryPattern);
+
+                    if (match.Groups.Count != 2)
+                    {
+                        App.Logger.WriteLine(LOG_IDENT, $"Failed to assert format for game join load time entry");
```

Comment:
Remove the '$'. $ is the identifier for a formatted string. You are not using formatting here.